cfn-resources/project-ip-access-list/docs/README.md

```diff
@@ -68,22 +60,24 @@ _Type_: <a href="listoptions.md">listOptions</a>
 
 _Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
 
-## Return Values
+#### Profile
 
-### Ref
+Profile used to provide credentials information, (a secret with the cfn/atlas/profile/{Profile}, is required), if not provided default is used
```

Comment:
Let's keep a standard text - from the encryption at rest PR, I think this works better:
```suggestion
The profile is defined in AWS Secret manager. See [Secret Manager Profile setup](../../../examples/profile-secret.yaml).
```